Formula/elasticsearch@7.6.rb

```diff
@@ -0,0 +1,150 @@
+class ElasticsearchAT76 < Formula
+  desc "Distributed search & analytics engine"
+  homepage "https://www.elastic.co/products/elasticsearch"
+  url "https://github.com/elastic/elasticsearch/archive/v7.6.2.tar.gz"
+  sha256 "6ff4871dcae6954e13680aefc196da574a59a36418d06a7e095550ce81a370f8"
+
+  head do
+    url "https://github.com/elasticsearch/elasticsearch.git"
+  end
```

Comment:
```suggestion
  sha256 "6ff4871dcae6954e13680aefc196da574a59a36418d06a7e095550ce81a370f8"
  head "https://github.com/elasticsearch/elasticsearch.git"
```

Reply:
Removed `head` (originally left it since it was in current `elasticsearch` formula)